yt_dlp/extractor/telemundo.py

```diff
@@ -0,0 +1,60 @@
+# coding=utf-8
+from __future__ import unicode_literals
+
+import json
+
+from .common import InfoExtractor
+from ..utils import (
+    try_get,
+    HEADRequest,
+    unified_timestamp
+)
+
+
+class TelemundoIE(InfoExtractor):
+
+    _VALID_URL = r'https?:\/\/(?:www\.)?telemundo\.com\/.+?(?=video)video\/[^\/]+(?P<id>tmvo\d{7})'
+    _TESTS = [{
+        'url': 'https://www.telemundo.com/noticias/noticias-telemundo-en-la-noche/empleo/video/esta-aplicacion-gratuita-esta-ayudando-los-latinos-encontrar-trabajo-en-estados-unidos-tmvo9829325',
+        'info_dict': {
+            'id': 'tmvo9829325',
+            'timestamp': 1621396800,
+            'title': 'Esta aplicación gratuita está ayudando a los latinos a encontrar trabajo en Estados Unidos',
+            'uploader': 'Telemundo',
+            'uploader_id': 'NBCU_Telemundo',
+            'ext': 'mp4',
+            'upload_date': '20210519',
+        },
+        'params': {
+            'skip_download': True,
+        }
+    }, {
+        'url': 'https://www.telemundo.com/shows/al-rojo-vivo/empleo/video/personajes-de-times-square-piden-que-la-ciudad-de-nueva-york-los-deje-volver-trabajar-tmvo9816272',
+        'only_matching': True,
+    }]
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
+        metadata = json.loads(
```

Comment:
use `self._parse_json`. You also might need to use `transform_source=js_to_json` if you are extracting from js and not json

Reply:
The `script`'s type is `application/json`, so `js_to_json` isn't needed.